test_regress/schedule.svg

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file was accidentally added I presume.

Reply:
Yep, sorry!